pkg/scheduler/plugins/elasticquota/plugin_test.go

```diff
@@ -1144,6 +1144,8 @@ func TestPlugin_DryRunPreemption(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			suit := newPluginTestSuitWithPod(t, tt.nodes, tt.pods)
 			pl := suit.plugin.(*Plugin)
+			pl.OnQuotaAdd(pl.generateSystemQuota())
```

Comment:
as we talked, we should create these two special Quota before starting Informer.